includes/offline-install-modifications.bash

```diff
@@ -15,7 +17,14 @@ apt-get --quiet install --download-only --yes libattr1-dev libc6 libstdc++6 \
   bash-completion htop curl wget multitail git util-linux bzip2 zip unzip \
   xz-utils software-properties-common man-db whiptail acl usbutils dirmngr \
   arping apt-transport-https bc sysstat jq moreutils avahi-daemon python3 \
-  python3-pip python3-wheel python3-setuptools avahi-autoipd fontconfig
+  python3-pip python3-wheel python3-setuptools avahi-autoipd fontconfig \
```

Comment:
I assume the other additions are related to the comitup hotspot?

Reply:
yes. comitup and its dependecies need to be available offline for an obvious reason

Reply:
👍